docs/01-get-started.md

```diff
@@ -1,16 +1,40 @@
 # Get started
 
-This page should give you an understanding of how a Serverpod project is structured, how you make calls to endpoints, and how you communicate with the database. Before going through it, make sure that you have the latest version of Serverpod installed. In the previous section, you can learn how to set up the Serverpod command line tools and install Serverpod Insights.
+This page will help you understand how a Serverpod project is structured, how to make calls to endpoints, and how to communicate with the database.
 
-## Creating a new Serverpod Mini project
+<div style={{ position : 'relative', paddingBottom : '56.25%', height : '0' }}><iframe style={{ position : 'absolute', top : '0', left : '0', width : '100%', height : '100%' }} width="560" height="315" src="https://www.youtube-nocookie.com/embed/FwttjcKyWFk" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></div>
 
-Serverpod Mini is a lightweight version of Serverpod that is perfect for small projects or when you want to try out Serverpod without setting up a full server. To get started with Serverpod Mini, read the [Getting started with Serverpod mini](https://medium.com/serverpod/getting-started-with-serverpod-mini-58c71ffefa54) guide.
+## Serverpod or Serverpod Mini?
 
-```bash
-$ serverpod create mypod --mini
-```
+Serverpod Mini is a lightweight version of Serverpod that is perfect for small projects or when you want to try out Serverpod without setting up a Postgres database. If you start with Mini, you can upgrade to the full version of Serverpod anytime.
+
+__[Get started with Mini](get-started-with-mini)__
+
+<details>
+<summary>__Serverpod vs Serverpod Mini comparison__</summary>
+<p>
+
+| Feature               | Serverpod | Serverpod Mini |
+|-----------------------|-----------|:--------------:|
+| Remote method calls   |     ✅     |        ✅       |
+| Generated data models |     ✅     |        ✅       |
+| Streaming data        |     ✅     |        ✅       |
+| Social authentication |     ✅     |                |
```

Comment:
Question: What does this mean? Serverpod Mini still supports authentication; it just doesn't support our authentication module.

You could, for example, build a JWT token validation authentication without having to store anything in the database.

This make it sound like social authentication is not possible with mini.

Reply:
Clarified.